app/models/invite.rb

```diff
@@ -66,7 +76,7 @@ def self.invite_by_email(email, invited_by, topic = nil, group_ids = nil, custom
       topic: topic,
       group_ids: group_ids,
       custom_message: custom_message,
-      send_email: true
+      emailed_status: emailed_status_types[:sending]
```

Comment:
It seems odd to create an invite with a status of `sending`. It would only seem appropriate to be `sending` to me if the email is queued for sending. At this point it makes more sense to be `pending.`

Reply:
`pending` status is reserved for bulk invites. If I set the status to `pending` here then invite email will not be sent instantaneously and since we are not queuing the `ProcessBulkInviteEmails` job here the email will be sent only after a bulk invite process is triggered.

Here is how I am using the `emailed_status` column for different types of invites (copied from PR description):

> For normal email invites, invite record is created with emailed_status
> set to 'sending'.
> 
> When bulk invites are sent invite record is created with emailed_status
> set to 'pending'.
> 
> For invites that generates link, invite record is created with
> emailed_status set to 'not required'.
> 
> Once the email is sent via InviteEmail job the invite emailed_status
> is updated to 'sent'.

Reply:
Logically it confuses me that if you want an email to be sent you would assign `sending` as the status because at that point it is not "sending".

I would suggest this:

Normal invites are created with "pending"

Bulk Invites are created with "bulk_pending"

When an email is being send it's "sending"

When an email has been sent it's "sent"

Reply:
Understood, will make it so.

Reply:
PR is updated.